src/ServiceControl/Operations/CheckRavenDBIndexErrors.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace ServiceControl
+{
+    using System;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+    using NServiceBus.CustomChecks;
+    using NServiceBus.Logging;
+    using Raven.Client;
+
+    class CheckRavenDBIndexErrors : CustomCheck
+    {
+        public CheckRavenDBIndexErrors(IDocumentStore store)
+            : base("Error Database Index Errors", "ServiceControl Health", TimeSpan.FromMinutes(5))
+        {
+            _store = store;
+        }
+
+        public override Task<CheckResult> PerformCheck()
+        {
+            var statistics = _store.DatabaseCommands.GetStatistics();
+            var indexes = statistics.Indexes.OrderBy(x => x.Name).ToArray();
+
+            if (statistics.Errors.Length <= 0)
```

Comment:
```suggestion
            if (statistics.Errors.Length = 0)
```

Nitpicking

Reply:
But then the code should at least compile ;) We are not writing javascript here ;)